src/translations/af-ZA.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
i made a task to handle integration of these translations into our `extract_intl` script in main: https://transcend.height.app/T-10388

Reply:
where are these translations being loaded? at a quick glance i dont see us importing `@transcend-io/internationalization` at all

Reply:
Good catch. I need to update this to use our new internationalization package

Reply:
@michaelfarrell76 I thought this tech debt? I vaguely remember Ben making a one-off translation thing

Reply:
o nvm i do see us loading this -- https://github.com/transcend-io/consent-manager-ui/blob/eb909ec079a685aca3940ebe690d5a9f666fe6dd/src/translations/index.ts#L2

Reply:
oh cool that looks good! i may have missed that in the package.json